frontend/package.json

```diff
@@ -60,7 +60,10 @@
 		"@opentelemetry/context-zone": "^1.10.0",
 		"@opentelemetry/exporter-trace-otlp-http": "^0.36.0",
 		"@opentelemetry/instrumentation": "^0.36.0",
+		"@opentelemetry/resources": "^1.10.1",
 		"@opentelemetry/sdk-node": "^0.36.0",
-		"@opentelemetry/sdk-trace-web": "^1.10.0"
+		"@opentelemetry/sdk-trace-web": "^1.10.0",
+		"@opentelemetry/semantic-conventions": "^1.10.1",
+		"svelte-intl-precompile": "^0.12.1"
```

Comment:
interesting :-), precompile?

Reply:
yeah, sadly it's not in the html, but the json file gets compiled into functions that render the text out, so at runtime the string isn't getting parsed for formatting and interpolation.